azkaban-common/src/main/java/azkaban/project/JdbcProjectLoader.java

```diff
@@ -462,27 +488,32 @@ private void uploadProjectFile(Connection connection, Project project,
         size = bufferedStream.read(buffer);
       }
     } catch (IOException e) {
-      throw new ProjectManagerException("Error chunking file " + filename);
+      throw new ProjectManagerException("Error chunking file " + localFile.getName());
```

Comment:
Good point. Will add it. I am not too worried since the chunking logic, etc will go away when we move to the new storage.